src/snowcli/cli/snowpark/procedure/commands.py

```diff
@@ -36,11 +36,13 @@
 
 
 @app.command("init")
-def procedure_init() -> None:
+@with_output
+def procedure_init() -> OutputData:
     """
     Initialize this directory with a sample set of files to create a procedure.
     """
     create_project_template("default_procedure")
+    return OutputData()
```

Comment:
Let's return some message, "Project initialised" or something

Reply:
For `OutputData()` we print `Done`. Is it ok?